docs/index.md

```diff
@@ -1,10 +1,35 @@
 # MCI | Industrial Engineering & Management
 
 
-???+ info ":fontawesome-solid-person-digging:"
+???+ info
     
-    This site is currently under construction. Be aware, the content can change
-    at any time.
+    This page hosts all course materials for *Mathematics & IT*.
```

Comment:
Maybe: This page hosts course materials for variouse python lectures.

In the future we could add bullet points for the courses and what chapters/parts are relevent for it.

Reply:
I've applied your suggestion. 😄

You're right, I guess in the future we might have to rework the landing page in general and restructure it a bit.